config/sandruinblacklist.txt

```diff
@@ -4,6 +4,7 @@
 #SPECIFYING AN INVALID MAP WILL RESULT IN RUNTIMES ON GAME START
 
 ##MISC
+#_maps/RandomRuins/SandRuins/whitesands_alien_facility.dmm
```

Comment:
This is a deprecated file that we do not upkeep. You should not make changes to it.